ddtrace/appsec/_ddwaf/waf_stubs.py

```diff
@@ -0,0 +1,174 @@
+from abc import ABC
+from abc import abstractmethod
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import Generic
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Type
+from typing import TypeVar
+from typing import Union
+
+from ddtrace.appsec._constants import DEFAULT
+from ddtrace.appsec._utils import _observator
+from ddtrace.internal.logger import get_logger
+
+
+LOGGER = get_logger(__name__)
+
+
+T = TypeVar("T")
+
+
+DDWafRulesType = Union[None, int, str, List[Any], Dict[str, Any]]
+
+
+class DDWaf_result:
```

Comment:
Isn't it more Pythonic to use `waf.pyi` instead of `waf_stubs` for the interfaces?